internal/ord/ord.go

```diff
@@ -171,9 +170,10 @@ func createInscriptionTxCtxData(net *chaincfg.Params, inscriptionRequest *Inscri
 	// to skip txscript.MaxScriptSize 10000
 	inscriptionScript = append(inscriptionScript, txscript.OP_ENDIF)
 
+	leafNode := txscript.NewBaseTapLeaf(inscriptionScript)
 	proof := &txscript.TapscriptProof{
-		TapLeaf:  txscript.NewBaseTapLeaf(schnorr.SerializePubKey(privateKey.PubKey())),
-		RootNode: txscript.NewBaseTapLeaf(inscriptionScript),
+		TapLeaf:  leafNode,
```

Comment:
only the `TapLeaf.LeafVersion` is used in `ToControlBlock`, so the original code does not cause error, but is misleading.